src/guides/v2.4/graphql/mutations/copy-items-between-requisition-list.md

```diff
@@ -0,0 +1,95 @@
+---
+group: graphql
+title: copyItemsBetweenRequisitionList mutation
```

Comment:
This mutation is now called `copyItemsBetweenRequisitionLists` (ends with an s). So much has changed in the schema for this mutation that it would be better to start over on this topic than try to point out all the differences. Note that the descriptions have been updated in the schema file. Use those where you can.

Reply:
Sure, will modify accordingly.